app/controllers/concerns/blacklight/facet.rb

```diff
@@ -20,9 +20,14 @@ def facets_from_request(fields = facet_field_names)
       fields.map { |field| facet_by_field_name(field) }.compact
     end
 
-    def facet_field_names(facet_type = :all)
+    def facet_field_names(requested_facet_type = :default)
+      requested_facet_type ||= :default
```

Comment:
this is unneeded - the declaration above will suffice to set the value in case this method is called with a Nil value

Reply:
hm; this actually isn't equivalent in the case where a `nil` arg is explicitly passed in (which, fwiw, I think may actually be likely to happen in practice?). I agree this is confusing and not ideal though -- let's remember to fold it into a more general discussion about how to handle `facet_type` attribute?

Reply:
you're right about that method param subtlety, but a `nil` facet_type is a code smell we should work to resolve, in time